### ISSUE TITLE
Order of XZX vector is 3-2-1 rather than 1-2-3?

### ISSUE THREAD

**Issue by andriyDev** [dilevin/computer-graphics-kinematics#22]
Is the order of the angle vector angle 3, angle 2, angle 1, or the reverse? When building the euler angle transformation, by the notes, it should be R_3 * R_2 * R_1, so by this logic the angle vector should be used as v.z(), v.y(), v.x(), but when I do this I can't quite make the knight "dab" (I didn't think I'd ever have to say that). Trying v.x(), v.y(), v.z() however it seems to make sense. Is this fine? Is this intended?

**Comment by chl9797** [comment]
Yeah, you’re right. The order of storage in Euler angle “xzx” seems to be 3-2-1.
But I don’t know why... Maybe we should fix this or add comments somewhere next time. Is this intended? @dilevin

**Comment by dilevin** [comment]
this is fine
